scheduler/scheduler-server/src/test/java/performancetests/verification/VerifyChangeTest.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package performancetests.verification;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.*;
+import java.util.*;
+
+import org.apache.log4j.Logger;
+import org.apache.log4j.PropertyConfigurator;
+import org.junit.Test;
+
+
+public class VerifyChangeTest {
+
+    static {
+        PropertyConfigurator.configure(VerifyChangeTest.class.getResource("/performancetests/config/log4j.properties"));
+    }
+
+    private static final Logger LOGGER = Logger.getLogger(VerifyChangeTest.class);
+
+    /**
+     * When any performance metric changes more or less than
+     * given threshold then this test fails
+     */
+    private static final Double THRESHOLD = Double.parseDouble(System.getProperty("threshold"));
+
+    @Test
+    public void changeShouldNotBeThanThreshold() throws IOException {
+
+        final List<File> twoLastFiles = getTwoLastFiles();
+
+        final File newFile = twoLastFiles.get(0);
+        final File previousFile = twoLastFiles.get(1);
+
+        final Map<String, Long> prediousReport = readReport(previousFile);
+        final Map<String, Long> newReport = readReport(newFile);
+
+        final Map<String, Double> compared = compareReports(prediousReport, newReport);
+
+        boolean toNotify = false;
+        String notifyReport = String.format("Two last performance reports were compared: \n\t\t'%s' \n\tand\n\t\t'%s'\n",
+                                            previousFile.getName(),
+                                            newFile.getName());
+        notifyReport += "Some performance metrics have been changed bigger than " + (THRESHOLD * 100) + "%:\n";
```

Comment:
"have been changed bigger than"you can change it to "have become bigger" or "the difference has become bigger than"depending what you like to say.